domain-ee/ee-ep-merge-app/src/python_src/model/merge_job.py

```diff
@@ -0,0 +1,36 @@
+from enum import Enum
+from typing import Any
+from uuid import UUID
+
+from pydantic import BaseModel, conint
+
+
+class JobState(str, Enum):
+    COMPLETED_ERROR = 'COMPLETED_ERROR'
+    PENDING = 'PENDING'
+    RUNNING_GET_PENDING_CLAIM_CONTENTIONS = 'RUNNING_GET_PENDING_CLAIM_CONTENTIONS'
+    RUNNING_GET_SUPP_CLAIM_CONTENTIONS = 'RUNNING_GET_SUPP_CLAIM_CONTENTIONS'
+    RUNNING_SET_TEMP_STATION_OF_JURISDICTION = 'RUNNING_SET_TEMP_STATION_OF_JURISDICTION'
+    RUNNING_MERGE_CONTENTIONS = 'RUNNING_MERGE_CONTENTIONS'
+    RUNNING_UPDATE_PENDING_CLAIM_CONTENTIONS = 'RUNNING_UPDATE_PENDING_CLAIM_CONTENTIONS'
+    RUNNING_CANCEL_SUPP_CLAIM = 'RUNNING_CANCEL_SUPP_CLAIM'
+    COMPLETED_SUCCESS = 'COMPLETED_SUCCESS'
+
+    def __str__(self):
+        return self.value
+
+
+class MergeJob(BaseModel):
+    job_id: UUID
+    pending_claim_id: conint(strict=True)
+    supp_claim_id: conint(strict=True)
```

Comment:
Could you rename `supp_claim` everywhere, in any variant, to something like `ep400_claim`? The "supp claim" label does get bandied around with EP 400s, but it's misleading; at VA, Supplemental Claims are a totally different beast, and we should try to reduce confusion rather than perpetuate it.

Reply:
Ah! Can do! It turns out I was a victim of the confusion as well!